src/UI/pages/Payments/WithdrawPaymentsModal.tsx

```diff
@@ -0,0 +1,130 @@
+import { Snackbar } from "@mui/material";
+import dayjs from "dayjs";
+import React, { useCallback, useEffect, useState } from "react";
+
+import PaymentsManager from "../../../Managers/PaymentsManager";
+import { IPayment } from "../../../models";
+import { formatCurrency } from "../../../Utils/functions/Ui";
+import { MoneyStye } from "../../components/Generic/Style";
+import ModalWrapper from "../../components/Modals/ModalWrapper";
+import TableWrapper from "../../components/Tables/TableWrapper";
+import useHome from "../Home/useHome";
+import { LabelSummaryIndicator, SummaryContainer, SummaryItem, ValueSummaryIndicator } from "./Style";
+
+interface ICreateUserModalProps {
+  isOpen: boolean;
+  onClose: (saved: boolean) => void;
+}
+interface IWithdrawsResponse {
+  successWithdraws: string[];
+  failedWithdraws: string[];
+}
+const WithdrawPaymentsModal: React.FC<ICreateUserModalProps> = ({ isOpen, onClose }) => {
+  const [fetchingLoading, setFetchLoading] = React.useState(false);
+  const [modalData, setModalData] = React.useState<any | undefined>();
+  const [showPopup, setShowPopup] = React.useState(false);
+  const [toastMeessage, setToastMessage] = useState("");
+  const [pendingWithdraws, setPendingWithdraws] = useState<IPayment[]>([]);
+  const [withdrawsResponse, setWithdrawsResponse] = useState<IWithdrawsResponse>({ successWithdraws: [], failedWithdraws: [] });
+  const { pendingWithdrawsTableColumns } = useHome();
+
+  const handleClodeModal = useCallback(() => {
+    if (!modalData) {
+      onClose(false);
+      return;
+    }
+
+    setModalData({ ...modalData, isOpen: false });
+    onClose(false);
+  }, [modalData]);
+
+  const initModalInfo = useCallback(() => {
+    setModalData({
+      id: "withdraw-payments-modal",
+      isOpen: false,
+      title: "Levantamento de Pagamentos",
+      actionButtonText: "Levantar",
+    });
+  }, [modalData]);
+
+  useEffect(() => {
+    initModalInfo();
+  }, []);
+
+  function fetchPendingWithdraws() {
+    setFetchLoading(true);
+    PaymentsManager.getPendingWithdraws()
+      .then((withdraws: IPayment[]) => {
+        setPendingWithdraws(withdraws);
+      })
+      .finally(() => {
+        setFetchLoading(false);
+      });
+  }
+
+  useEffect(() => {
+    fetchPendingWithdraws();
+  }, []);
+
+  const handleClose = (event: React.SyntheticEvent | Event, reason?: string) => {
+    if (reason === "clickaway") {
+      return;
+    }
+    setShowPopup(false);
+  };
+
+  const createPaymentData = useCallback(
+    (pendingWithdraws: IPayment[]): any[] => {
+      if (!pendingWithdraws) {
+        return [];
+      }
+
+      return pendingWithdraws.map((data: IPayment) => ({
+        order: `${data.orderNumber}`,
+        amount: <MoneyStye mode="SUCCESS">{formatCurrency(data.price)}</MoneyStye>,
+        date: dayjs(data.createdAt).format("DD/MM/YYYY HH:mm"),
```

Comment:
We already have date format inside Constants.ts

Reply:
solved